tvm/python/tvm/topi/adreno/conv2d.py

```diff
@@ -582,4 +582,4 @@ def copy_to_texture(stage):
         KHKW = KH*KW
 
     if isinstance(N, int):
-        cfg.add_flop(2 * N * OH * OW * OCC * OCB * KHKW)
+        cfg.add_flop(2 * N * OH * OW * OCC * OCB * KHKW)
```

Comment:
No newline at end of file. Please correct it

Reply:
Done